20151222/rails-5.0.0.beta1-demo/test/controllers/comments_controller_test.rb

```diff
@@ -0,0 +1,53 @@
+require 'test_helper'
+
+class CommentsControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @post = posts(:one)
+    @comment = comments(:one)
+  end
+
+  test "should get index" do
+    get comments_url(post_id: @post)
+    assert_response :success
+  end
+
+  test "should get new" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.